index.html

```diff
@@ -31,24 +30,21 @@ <h1>Please choose the ingredients you wish to have included:</h1>
         </section><!--Search Button-->
 
         <section><!--Random recipe generator button-->
-            <button type="button"
+            <button type="button" id="randomSearchBtn"
                 class="text-white bg-gradient-to-br from-purple-600 to-blue-500 hover:bg-gradient-to-bl focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2">Random
                 Search</button>
             
         </section>
        
-        <section><!--recipe save button-->
-            <button type="button"
-            class="text-white bg-gradient-to-br from-purple-600 to-blue-500 hover:bg-gradient-to-bl focus:ring-4 focus:outline-none focus:ring-blue-300 dark:focus:ring-blue-800 font-medium rounded-lg text-sm px-5 py-2.5 text-center me-2 mb-2">save</button>
-        
-         </section>
+    
         
         <section class="columns-1">
             <div id="drink-options">
                 <!--js rendered drink options render here-->
             </div>
         </section><!--section for rendered recipe list-->
     </main>
+    <link rel="save.html" href="save.html"
```

Comment:
We can remove these links between each page, they are braking the page and are not actually needed anymore